test/test_torch.py

```diff
@@ -4849,6 +4849,54 @@ def test_diff(self, device, dtype):
             scalar = torch.tensor(2, device=device, dtype=dtype)
             torch.diff(scalar)
 
+    def _test_histogram_numpy(self, t, bins, weights, density):
+        # Helper for test_histogram to compare with NumPy reference implementation
+        def to_np(t):
+            if t is None:
+                return None
+            if t.dtype == torch.bfloat16:
```

Comment:
Good accounting for the bfloat16 datatype; it's a pain that NumPy doesn't support it